src/App.test.tsx

```diff
@@ -1,74 +1,49 @@
-import { fireEvent, render, screen, act } from '@testing-library/react';
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import {
+  fireEvent, render, screen, act,
+} from '@testing-library/react';
 import App from './App';
 
-test('renders SenderForm',() => {
-  //Arrange
-  render(<App />)
-  //Act
-  const element =  screen.getByTestId('sender-form')
-  //Assert
-  expect(element).toBeInTheDocument()
-})
+test('renders SenderForm', () => {
+  // Arrange
+  render(<App />);
+  // Act
+  const element = screen.getByTestId('sender-form');
+  // Assert
+  expect(element).toBeInTheDocument();
+});
 
 test('renders form label', () => {
-  //Arrange
-  render(<App />)
-  //Act
-  const element = screen.getByText(/Secrets/)
-  //Assert
-  expect(element).toBeInTheDocument()
-  
-})
-
-test('render secret text box', () => {
-  //Arrange
-  render(<App/>)
-  const element = screen.getByTestId('textbox')
-  //Assert
-  expect(element).toBeInTheDocument()
-})
+  // Arrange
+  render(<App />);
+  // Act
+  const element = screen.getByText(/Secrets/);
+  // Assert
+  expect(element).toBeInTheDocument();
+});
+
+test('renders secret text box', () => {
+  // Arrange
+  render(<App />);
+  const element = screen.getByTestId('textbox');
+  // Assert
+  expect(element).toBeInTheDocument();
+});
 
 test('renders submit button', () => {
-  //Arrange
-  render(<App/>)
-  const element = screen.getByTestId('submit-button')
-  //Assert
-  expect(element).toBeInTheDocument()
-})
+  // Arrange
+  render(<App />);
+  const element = screen.getByTestId('submit-button');
```

Comment:
Just for the consistence.
```suggestion
  // Act
  const element = screen.getByTestId('submit-button');
```